tools/objc/interface.py

```diff
@@ -66,20 +66,20 @@ def __load_interfaces(self, config, package, type):
     @staticmethod
     def __find_class_hierarchy(config, package, type):
         base = "GlyCommon"
```

Comment:
base = config.data["base"]
Something like that... not required... just seemed like it would fit in with what you were trying to do.